src/Core/TileMesh.js

```diff
@@ -204,38 +202,13 @@ TileMesh.prototype.changeSequenceLayers = function changeSequenceLayers(sequence
 };
 
 TileMesh.prototype.getCoordsForLayer = function getCoordsForLayer(layer) {
-    if (layer.protocol.indexOf('wmts') == 0) {
```

Comment:
for now, I'm writing two static functions

`getCoordsForLayer (layer, node)` and `getZoomForLayer (layer, node)`

this will avoid contaminating the `preprocessDataLayer` which should only affect the properties of the source (WMTS, WMS...)